frontend/packages/console-shared/src/components/dynamic-form/types.ts

```diff
@@ -1,4 +1,4 @@
-export enum SchemaType {
+export enum JSONSchemaType {
```

Comment:
More descriptive name since 'schema' could mean UI schema or JSON schema in the context of the DynamicForm component.